draft-ietf-httpbis-priority.md

```diff
@@ -630,7 +630,7 @@ left as an implementation decision.
 
 Absence of a priority parameter in an HTTP response indicates the server's
 disinterest in changing the client-provided value. This is different from the
-logic being defined for the request header field, in which omission of a
+approach for the request header field, in which omission of a
```

Comment:
Take-it-or-leave-it, but maybe: "This is different from that in the request header field".

Reply:
or just "This is different from the request header field"

Reply:
That SGTM.

Reply:
done in #fc5eb16